doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,950 @@
+*panes*.
+
+### Streamlit Matplotlib Example
+
+```python
+import numpy as np
+import streamlit as st
+
+import matplotlib.pyplot as plt
+
+data = np.random.normal(1, 1, size=100)
+fig, ax = plt.subplots(figsize=(8,4))
+ax.hist(data, bins=20)
+
+st.pyplot(fig)
+```
+
+The app looks like
+
+![Streamlit Matplotlib Example](https://user-images.githubusercontent.com/42288570/243348167-4d2a3c36-59dc-4df5-b266-84dd1837f2b6.png)
+
+### Panel Matplotlib Example
+
+You will find Panels output *panes* in `pn.pane` module.
+
+```python
+import panel as pn
+import numpy as np
+
+from matplotlib.figure import Figure
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+data = np.random.normal(1, 1, size=100)
+fig = Figure(figsize=(8,4))
+ax = fig.subplots()
+ax.hist(data, bins=20)
+
+pn.pane.Matplotlib(fig, format='svg', sizing_mode='scale_both').servable()
+```
+
+We use Matplotlibs `Figure` interface instead of the `pyplot` interface to
+avoid memory leaks if you forget to close the figure. This is all described in the
+[Matplotlib Guide](https://panel.holoviz.org/reference/panes/Matplotlib.html).
+
+The app looks like
+
+![Panel Matplotlib Example](https://user-images.githubusercontent.com/42288570/243348449-233d1dcc-fdd3-4907-9085-50bf0772065f.png)
+
+### Output Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_object` function with the corresponding Panel
+`pn.pane.SomeObject` class.
+
+You can identify the corresponding Panel *pane* in the
+[Panes Section](https://panel.holoviz.org/reference/index.html#panes) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
+## Inputs
+
+In Panel the objects that can provide you with input values from users are called *widgets*.
+
+Panel provides widgets similar to the ones you know from Streamlit and some unique ones in addition.
+
+Check out the
+[Widgets Section](https://panel.holoviz.org/reference/index.html#widgets) of
+the [Component Gallery](https://panel.holoviz.org/reference/index.html) for the full list of
+*widgets*.
+
+### Streamlit Integer Slider Example
+
+```python
+import streamlit as st
+
+bins = st.slider(value=20, min_value=10, max_value=30, step=1, label="Bins")
+
+st.write(bins)
+```
+
+The app looks like
+
+![Streamlit Widgets Example](https://user-images.githubusercontent.com/42288570/243349378-a27fa7bd-b8dc-4b30-85f0-684a74ec40cf.png)
+
+### Panel Integer Slider Example
+
+You will find Panels input *widgets* in `pn.widgets` module.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+bins = pn.widgets.IntSlider(value=20, start=10, end=30, step=1, name="Bins").servable()
+
+pn.pane.Str(bins).servable()
+```
+
+If you debug and inspect the code you will notice a big difference. Streamlits `bins` value is an
+`integer` while Panels `bins` value is an `IntSlider`. This is the first real indication that
+Streamlit and Panel works in very different ways.
```

Comment:
I think this can be clarified further:
```suggestion
If you check the type of the variables, you will notice a key difference. Streamlit's `bins` returns the value of the slider as an `integer` while Panel's `bins` returns an `IntSlider`!

To access the value of the slider in Panel, you would need to call `bins.value`.
```